test-assets/role-manifests/bosh-run-bad-cpu.yml

```diff
@@ -5,3 +5,6 @@ roles:
   run:
     memory: 1
     virtual-cpus: -2
+configuration:
+  templates:
+    properties.tor.hostname: '((IP_ADDRESS))((DNS_RECORD_NAME))'
```

Comment:
I think if we mark the builtins as `Internal` it all works out.  You can't (from a role manifest) create an internal environment variable, but you can do so from code just fine.